test/testdata/resolver/abstract_validation.rb

```diff
@@ -92,6 +92,25 @@ class SplatChild2 < SplatParent
   def foo(**opts); end # error: Implementation of abstract method `SplatParent#foo` must accept *`args`
 end
 
+# https://github.com/sorbet/sorbet/issues/1215
```

Comment:
You're welcome to leave comments like this in the tests, but you don't have to.

If we need to recover the link, we can always use the `git log` or `git blame` to find the original PR and any tickets it links.